### ISSUE TITLE
Installation error w/ Packer v0.8.1

### ISSUE THREAD

**Issue by robwilkerson** [lmars/packer-post-processor-vagrant-s3#11]
$ go get github.com/lmars/packer-post-processor-vagrant-s3
    # github.com/lmars/packer-post-processor-vagrant-s3
    ../../../go/src/github.com/lmars/packer-post-processor-vagrant-s3/post-processor.go:31: undefined: packer.ConfigTemplate

This might be a change in the Packer core as indicated in [this ticket](https://github.com/schubergphilis/packer-cloudstack/issues/17) on another post processor. Hopefully this  is reasonably easy to fix.

**Comment by warnergodfrey** [comment]
I have the same problem.

**Comment by amandatory** [comment]
Same problem using the packer-post-processor-ovftool.

**Comment by yoheimuta** [comment]
I could reproduce it with packer v0.8.0 ~.

Installation with packer v0.7.5 is ok.
I show a workaround below. I don't try using this binary with packer v0.8.0 ~ yet (I tried only installation).

```ruby
# reproduce
[19:28:52 great@Mac-158 tmp ]$ export GOPATH=/mtburn/dev/go/tmp/
[19:29:07 great@Mac-158 tmp ]$ echo $GOPATH
/mtburn/dev/go/tmp/
[19:29:11 great@Mac-158 tmp ]$ go get github.com/lmars/packer-post-processor-vagrant-s3
# github.com/lmars/packer-post-processor-vagrant-s3
src/github.com/lmars/packer-post-processor-vagrant-s3/post-processor.go:32: undefined: packer.ConfigTemplate

# checkout v0.7.5
[19:29:11 great@Mac-158 tmp ]$ cd src/github.com/mitchellh/packer/
[19:33:08 great@Mac-158 packer master]$ git co v0.7.5

# get
[19:34:04 great@Mac-158 tmp ]$ go get github.com/lmars/packer-post-processor-vagrant-s3
[19:34:40 great@Mac-158 tmp ]$

# sounds good
[19:35:06 great@Mac-158 tmp ]$ ./bin/packer-post-processor-vagrant-s3
panic: Please do not execute plugins directly. Packer will execute these for you.
...
```

**Comment by lmars** [comment]
Fixed in #13.

**Comment by robwilkerson** [comment]
For whatever its worth to future searchers, I had to remove the existing source code that was downloaded before this would install properly:

    $ rm -rf ~/go/src/github.com/lmars/packer-post-processor-vagrant-s3
